packages/components/addon/components/hds/table/index.js

```diff
@@ -20,11 +20,21 @@ export default class HdsTableIndexComponent extends Component {
   /**
    * @param isStriped
    * @type {boolean}
-   * @default true
-   * @description Determines whether the table rows should have alternating background colors; defaults to true.
+   * @default false
+   * @description Determines whether the table rows should have alternating background colors; defaults to false.
    */
   get isStriped() {
-    return this.args.isStriped ?? true;
+    return this.args.isStriped ?? false;
+  }
+
+  /**
+   * @param isFixed
```

Comment:
For consideration: should we make it more explicit what it refers to and call it `isFixedLayout`?